sksparse/cholmod.pyx

```diff
@@ -31,8 +31,6 @@
 
 #cython: binding = True
 #cython: language_level = 3
-#distutils: include_dirs = NP_GET_INCLUDE /usr/include/suitesparse
```

Comment:
Debian installs the headers to /usr/include/suitesparse (but Arch Linux to /usr/include) so you need to keep the former option too.